dotnet/src/Agents/Abstractions/Definition/AgentInput.cs

```diff
@@ -39,16 +47,21 @@ public sealed class AgentInput
     public string? JsonSchema { get; set; }
 
     /// <summary>
-    /// Gets or sets a value indicating whether to handle the input value as potential dangerous content.
+    /// Gets or sets a value indicating whether the input can emit structural text.
     /// </summary>
     /// <remarks>
     /// The default is true.
-    /// When set to false the value of the input is treated as safe content.
+    /// When set to false the value of the input is treated as safe content i.e. the input can emit structural text.
     /// </remarks>
     public bool Strict { get; set; } = true;
 
     /// <summary>
     /// Gets or sets a sample value for the input.
     /// </summary>
+    /// <remarks>
+    /// This is used to provide examples to the user of the agent.
+    /// They can also be used to provide examples to the tooling that will be used to load and execute the agent.
+    /// If this section is not included, the runtime will use the default value for the input.
```

Comment:
```suggestion
    /// <remarks>
    /// This is used to provide an example to the user of the agent.
    /// This can also be used to provide an example to the tooling that will be used to load and execute the agent.
    /// If this section is not included, the runtime will use the default value for the input.
```

I'm not sure I understand the third sentence there.  Surely a sample should be irrelevant as to whether a default value is used by the caller or not?  Shouldn't the default value be based on whether a value was provided or not?